spec/support/shared/functional/file_resource.rb

```diff
@@ -366,7 +366,6 @@ def real_file?(file_path)
   include_context "diff disabled"
 
   before do
-    Chef::Log.level = :info
```

Comment:
This seemed to be the culprit for all the log spam running the functional tests locally.

Reply:
yep, global state is horrible